tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -282,6 +282,18 @@ void test_two_index_constraint_analytic(
   const auto& d_gauge_function =
       get<Tags::deriv<GaugeH, tmpl::size_t<3>, Frame::Inertial>>(gh_derivs);
 
+  // Populate spacetime derivative of gauge function
```

Comment:
Looks like you are assuming that the time derivative of H is zero.  There should be a comment indicating this.

Reply:
ah yes, that was because time derivs of H were not being used (or at least shouldn't be) in constraints' computation. I've changed that to actually populating them with integers to ensure that remains the case.